README.md

```diff
@@ -200,9 +200,14 @@ sudo apt install make gcc ripgrep unzip git neovim
 ```
 sudo apt update
 sudo apt install make gcc ripgrep unzip git
-echo "deb https://deb.debian.org/debian unstable main" | sudo tee -a /etc/apt/sources.list
-sudo apt update
-sudo apt install -t unstable neovim
+
+# Now we install nvim
+curl -LO https://github.com/neovim/neovim/releases/latest/download/nvim-linux64.tar.gz
```

Comment:
On a fresh debian install I get an error that curl is not installed:
```
vboxuser@debian125:~$ curl -LO https://github.com/neovim/neovim/releases/latest/download/nvim-linux64.tar.gz
bash: curl: command not found
```
so please add also `curl` to the above list of dependencies:
```
sudo apt install make gcc ripgrep unzip git curl
```